### PR TITLE
[QMR] New Measure - AAB-CH

### PR DESCRIPTION
**Pull request by ajaitasaini**
### Related ticket(s)

MDCT-2425

---
### How to test

1. Log into stateuser4
2. Add a child core set in 2023
3. Open the child core set and open AAB-CH
4. Match with ticket specs

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Comment by ajaitasaini** [comment]
**going to wait till Aileen's changes in [this PR](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/pull/1764) merge with master to make sure my changes don't cause merge conflicts

**Review by karla-vm — DISMISSED** [review]
LGTM!

**Review by keeysnc — APPROVED** [review]
LGTM!

File: services/ui-src/src/measures/2023/rateLabelText.ts [hunk 1]
```diff
@@ -22,6 +22,16 @@ export const data = {
...
     },
+    "AAB-CH": {
+        "qualifiers": [
+            {
+                "label": "Ages 3 months to 17 years",
+                "text": "Ages 3 months to 17 years",
+                "id": "xS5HMm"
+            },
+        ],
+        "categories": []
```
Comment (ailZhou) [inline services/ui-src/src/measures/2023/rateLabelText.ts:33]:
```suggestion
        "categories": [{"id":"ZCy3XP", "label": "", "text":""}]
```

slight change needed due to the label update, all categories need to have an id

**Review by ailZhou — APPROVED** [review]
Looks Good!

**Review by keeysnc — APPROVED** [review]
LGTM!

**Pull request merged**